servers/audio_server.h

```diff
@@ -297,6 +297,8 @@ class AudioServer : public Object {
 
 	SafeList<AudioStreamPlaybackListNode *> playback_list;
 	SafeList<AudioStreamPlaybackBusDetails *> bus_details_graveyard;
+	void _delete_stream_playback(Ref<AudioStreamPlayback> p_playback);
+	void _delete_stream_playback_list_node(AudioStreamPlaybackListNode *p_node);
```

Comment:
Nitpick, but usually we define private methods after the block of private variables.

Reply:
I suggest that we merge this PR, then I'll create a new one reorganizing the mess that is the `AudioServer` class.